pkgs/applications/misc/coffeegrindsize/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, python3, fetchFromGitHub }:
+let
+  python = python3.override {
+    # for tkinter support
+    x11Support = true;
+  };
+in
```

Comment:
This is not really needed. You already include tkinter in the package definition below.

Reply:
Thanks for catching that! I was struggling a bit when packaging it to make the GUI work